### ISSUE TITLE
 No module named 'conda._vendor.auxlib' on windows when using GIT_DESCRIBE_TAG

### ISSUE THREAD

**Issue by johanvdw** [conda/conda-build#4334]
### Actual Behavior

I use GIT_DESCRIBE_TAG this version in my meta.yaml:

```
package:
  name: cn_ws
  version: {{ GIT_DESCRIBE_TAG }}
```

When I try to run conda build conda to build my package on windows, I get the error:

```
==> C:\Program Files\Git\cmd\git.exe describe --tags --dirty <==

4.2.0-20-g6184871

==> C:\Program Files\Git\cmd\git.exe status <==

On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean

Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\environ.py:444: UserWarning: The environment variable 'DRONE_BUILD_NUMBER' is being passed through with value '0'.  If you are splitting build and test phases with --no-test, please ensure that this value is also set similarly at test time.
  warnings.warn(
Traceback (most recent call last):
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\Scripts\conda-build-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\cli\main_build.py", line 480, in main
    execute(sys.argv[1:])
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\cli\main_build.py", line 469, in execute
    outputs = api.build(args.recipe, post=args.post, test_run_post=args.test_run_post,
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\api.py", line 186, in build
    return build_tree(
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\build.py", line 3062, in build_tree
    metadata_tuples = render_recipe(recipe, config=cfg, variants=variants,
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\render.py", line 860, in render_recipe
    rendered_metadata = distribute_variants(m, variants,
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\render.py", line 756, in distribute_variants
    if mv.needs_source_for_render and mv.variant_in_source:
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\metadata.py", line 1870, in variant_in_source
    self.parse_again(permit_undefined_jinja=True, allow_no_other_outputs=True,
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\metadata.py", line 1006, in parse_again
    self.meta = parse(self._get_contents(permit_undefined_jinja,
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\metadata.py", line 1573, in _get_contents
    env.globals.update(environ.get_dict(m=self, skip_build_id=skip_build_id))
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\environ.py", line 280, in get_dict
    d.update(meta_vars(m, skip_build_id=skip_build_id))
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\environ.py", line 483, in meta_vars
    d.update(get_git_info(git_exe, git_dir, meta.config.debug))
  File "C:\Users\johan\miniconda3-newer\envs\cnws-build\lib\site-packages\conda_build\environ.py", line 191, in get_git_info
    from conda._vendor.auxlib.packaging import _get_version_from_git_tag
ModuleNotFoundError: No module named 'conda._vendor.auxlib'
```

### Expected Behavior

The same meta.yml  builds fine on Linux.

### Steps to Reproduce

1. Clean install of miniconda
2. Create an environment containing just conda build:
`conda create --name cnws-build conda-build`
3. run conda-build 

##### `Output of conda info`

```
(cnws-build) C:\Users\johan\git\cn_ws>conda info

     active environment : base
    active env location : C:\Users\johan\miniconda3-newer\envs\cnws-build
            shell level : 2
       user config file : C:\Users\johan\.condarc
 populated config files : C:\Users\johan\.condarc
          conda version : 4.11.0
    conda-build version : 3.21.6
         python version : 3.10.0.final.0
       virtual packages : __win=0=0
                          __archspec=1=x86_64
       base environment : C:\Users\johan\miniconda3-newer\envs\cnws-build  (writable)
      conda av data dir : C:\Users\johan\miniconda3-newer\envs\cnws-build\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\johan\miniconda3-newer\envs\cnws-build\pkgs
                          C:\Users\johan\.conda\pkgs
                          C:\Users\johan\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\johan\miniconda3-newer\envs\cnws-build\envs
                          C:\Users\johan\.conda\envs
                          C:\Users\johan\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.11.0 requests/2.26.0 CPython/3.10.0 Windows/10 Windows/10.0.17763
          administrator : False
             netrc file : None
           offline mode : False
```

**Comment by JanuszL** [comment]
It seems to be caused by https://github.com/conda/conda/pull/11025.

**Comment by JanuszL** [comment]
It seems that https://github.com/conda/conda-build/pull/4335 should fix that.

**Comment by johanvdw** [comment]
I closed, as it indeed seems a duplicate issue.

**Comment by johanvdw** [comment]
Before this gets merged: fixing conda to 4.10.3 solved the issue for me.
